### DISCUSSION TITLE
Request release to support installing minified css

### DISCUSSION THREAD

**Discussion by jeromecovington** [PrismJS/prism#3289] [Q&A]
It looks like the minified css files were added here https://github.com/PrismJS/prism/pull/3073 on October 4 but the last publish of prismjs@1.25.0 was on September 16. Possible for us to get a new publish?

**Comment by RunDevelopment** [comment]
I'll make a new release until the end of the week. (For more info, see #3290.)

**Reply by RunDevelopment** [reply]
Just published 1.26.0.
